app/models/nic/base.rb

```diff
@@ -81,18 +81,34 @@ class Jail < ::Safemode::Jail
       allow :managed?, :subnet, :subnet6, :virtual?, :physical?, :mac, :ip, :ip6, :identifier, :attached_to,
         :link, :tag, :domain, :vlanid, :mtu, :bond_options, :attached_devices, :mode,
         :attached_devices_identifiers, :primary, :provision, :alias?, :inheriting_mac,
-        :children_mac_addresses, :nic_delay, :fqdn, :shortname
+        :children_mac_addresses, :nic_delay, :fqdn, :shortname, :type, :managed?, :bond?, :bridge?, :bmc?
     end
 
     # include STI inheritance column in audits
     def self.default_ignored_attributes
       super - [inheritance_column]
     end
 
+    def managed?
+      type == 'Nic::Managed'
```

Comment:
This is conflicting with managed boolean column. Nic::Managed class says "regular physical interface", read eth0. This would cause a regression, since `managed?` today is checked for whether we run orchestration on this NIC. Please rename to regular? or something similar

Reply:
Good catch, I actually only need to add this to safemode, since it exists already, I am going to remove this method and keep it in safemode allowed list.